src/HPRT/toolkit/models.py

```diff
@@ -316,4 +310,36 @@ def get_labels(self):
         return labels
 
 
-        
+class GeneralHealth(models.Model):
+    patient = models.ForeignKey(Patient, on_delete = models.CASCADE)
+    doctor = models.ForeignKey(Doctor, on_delete = models.CASCADE)
+    date = models.DateField()
+    gh1 = models.IntegerField(null=True, choices=[('',"Choose one"),(1,"1 = Poor"),(2,"2 = Fair"),(3,"3 = Good"),(4,"4 = Very Good"),(5,"5 = Excellent")])
+    gh2 = models.IntegerField(null=True, choices=[('',"Choose one"),(1,"1 = None"),(2,"2 = A Little"),(3,"3 = Some"),(4,"4 = Quite A Lot"),(5,"5 = Very Much")])
+    gh3 = models.IntegerField(null=True, choices=[('',"Choose one"),(1,"1 = Extremely"),(2,"2 = Quite A Lot"),(3,"3 = Moderately"),(4,"4 = Slightly"),(5,"5 = Not At All")])
+    gh4 = models.IntegerField(null=True, choices=[('',"Choose one"),(1,"1 = Seven"),(2,"2 = Five to Seven"),(3,"3 = Three to Five"),(4,"4 = One to Three"),(5,"5 = Zero")])
+    gh5 = models.IntegerField(null=True, choices=[('',"Choose one"),(1,"1 = None"),(2,"2 = Less than Four"),(3,"3 = Four to Five"),(4,"4 = Five to Seven"),(5,"5 = More than Seven")])
+    gh6 = models.IntegerField(null=True, choices=[('',"Choose one"),(1,"1 = Daily"),(2,"2 = Weekly"),(3,"3 = Monthly"),(4,"4 = Yearly"),(5,"5 = No")])
+    gh7 = models.IntegerField(null=True, choices=[('',"Choose one"),(1,"1 = Zero"),(2,"2 = One to Three"),(3,"3 = Three to Five"),(4,"4 = Five to Seven"),(5,"5 = More than Seven")])
+    gh8 = models.IntegerField(null=True, choices=[('',"Choose one"),(1,"1 = Zero"),(2,"2 = One to Three"),(3,"3 = Three to Five"),(4,"4 = Five to Seven"),(5,"5 = More than Seven")])
+    gh9 = models.IntegerField(null=True, choices=[('',"Choose one"),(1,"1 = No"),(2,"2 = Yes")])
+    gh10 = models.IntegerField(null=True, choices=[('',"Choose one"),(1,"1 = No"),(2,"2 = No, but trying to obtain work"),(3,"3 = Yes")])
+    gh11 = models.IntegerField(null=True, choices=[('',"Choose one"),(1,"1 = Yes"),(2,"2 = Yes and has been referred"),(3,"3 = No")])
+    gh12 = models.IntegerField(null=True, choices=[('',"Choose one"),(1,"1 = Yes"),(2,"2 = Yes and has been referred"),(3,"3 = No")])
+    score = models.DecimalField(max_digits=3, decimal_places=1)
+
+    def __iter__(self):
```

Comment:
Docstring here - what does this do?

Reply:
No clue. It was there for the other models so I left it.